README.md

```diff
@@ -141,7 +141,7 @@ In addition to the standard connection parameters the driver supports a number o
 | readOnly                      | Boolean | true    | Puts this connection in read-only mode |
 | disableColumnSanitiser        | Boolean | false   | Enable optimization that disables column name sanitiser |
 | assumeMinServerVersion        | String  | null    | Assume the server is at least that version |
-| currentSchema                 | String  | null    | Specify the schema to be set in the search-path |
+| currentSchema                 | String  | null    | Specify the schema (or several schema separated by comma's) to be set in the search-path |
```

Comment:
@davecramer , am I right it should read `commas` not `comma's`?

Reply:
commas would be correct, yes.

Reply:
fixed